src/odemis/gui/cont/acquisition.py

```diff
@@ -1395,6 +1397,13 @@ def update_acquisition_time(self):
             lvl = logging.WARN
         else:
             streams = self._tab_data_model.acquisitionStreams
+            folds = tuple(acqmng.foldStreams(self._tab_data_model.acquisitionStreams))
+            if len(streams) > len(folds):
+                self.lbl_fold_acq.Show()
+                self.bmp_fold_acq_info.Show()
+            else:
+                self.lbl_fold_acq.Hide()
+                self.bmp_fold_acq_info.Hide()
```

Comment:
You can write it this way:
```suggestion
            has_folds = len(streams) >  len(acqmng.foldStreams(streams))
            self.lbl_fold_acq.Show(has_folds)
            self.bmp_fold_acq_info.Show(has_folds)
```